suite-native/receive/src/components/DevicePaginationButton.tsx

```diff
@@ -15,7 +15,6 @@ type DeviceScreenPaginationProps = {
     onPress: () => void;
 };
 
-const T2B1_BUTTON_COLOR = '#2B2D2B';
```

Comment:
please do not remove this const, it's not clear what's is that value now

Reply:
I put it back in 7552647f58de52aa1bcd03aa3f568118b034bb8a, but the naming is quite weird, it says T2B1, but is not used for T2B1 🤷